GDJS/Runtime/pixi-renderers/runtimegame-pixi-renderer.ts

```diff
@@ -592,7 +592,7 @@ namespace gdjs {
             ? gdjs.InputManager.MOUSE_RIGHT_BUTTON
             : e.button === 1
             ? gdjs.InputManager.MOUSE_MIDDLE_BUTTON
-            : gdjs.InputManager.MOUSE_LEFT_BUTTON
+            : e.button
```

Comment:
I feel like this logic should be in a dedicated function to explain why this is done like this and why we're not just using e.button

Reply:
agreed. This could be re-used above as well.
The code is confusing as it sometimes remap the code and sometimes doesn't.
So if it works well, let's put it inside a function

Reply:
done!